CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.10.0
+
+- Replaced depreciated `Endpoint` with endpoint_url, disabled use of `Endpoint 2.0`
+- Updated `aws-config ` to 0.54.1
+- Updated `aws-sdk-athena` to 0.24.0
+- Updated `aws-sdk-s3` to 0.24.0
+- Updated `aws-sdk-sqs` to 0.24.0
+- Updated `aws-smithy-http` 0.54.4
+- Updated `aws-types` to 0.54.1
+- Updated `rust crate tokio` to 1.26.0
+- Updated `rust crate clap` to 4.1.8
+- Updated `rust crate bytesize` to 1.2.0
+- Updated `rust crate serial_test` to v1
+- Updated `localstack docker tag` to 1.4.0
+- Updated `python docker tag` to v3.11
```

Comment:
**nitpick**: The docker tag dependencies are purely for our internal test and dev setup, and don't affect users of `cobalt-aws` in any way.  So they won't care about what versions we're using, and we shouldn't add things they won't care about to the changelog.